packages/core-components/src/layout/SignInPage/SignInPage.tsx

```diff
@@ -167,6 +169,11 @@ export const SingleSignInPage = ({
           profile,
         }),
       );
+      analytics.captureEvent('signIn', 'success', {
+        attributes: {
+          userEntityRef: identityResponse.identity.userEntityRef,
+        },
+      });
     } catch (err: any) {
       // User closed the sign-in modal
       setError(err);
```

Comment:
Should there be also analytics for sign in errors at the same time?

Reply:
I'm not sure that we should be submitting any attributes that contain any PII.

Looking at some of the modules that do analytics, they already take care of passing a `userId` attribute. 

https://github.com/backstage/community-plugins/blob/main/workspaces/analytics/plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts#L280-L291